src/user/controller/by-href.ts

```diff
@@ -40,6 +40,7 @@ class UserByHrefController extends Controller {
           isAdmin,
           await groupService.findGroupsForPrincipal(principal),
         );
+        ctx.redirect(303, principal.href);
```

Comment:
With this addition, we can skip the rendering part (ctx.response.body doesn't need to be set).

Also, this change should probably just be applied to all 3 principal types... With that, I think the `switch` statement can actually be removed?